README.md

```diff
@@ -1,9 +1,19 @@
 # Pyarc2stac
 
+A reverse proxy that dataset metadata from ESRI data services (ImageServer, MapServer, FeatureServer, SomethingServer) as [STAC](https://stacspec.org/), 
```

Comment:
The reverse proxy is actually [arc2stac](https://github.com/NASA-IMPACT/arc2stac). This repos is more for creating STAC collection using ArcGIS metadata. @amarouane-ABDELHAK can you confirm if this is correct?

Reply:
Yes [arc2stac](https://github.com/NASA-IMPACT/arc2stac) is the reverse proxy